modeltestSDK/api.py

```diff
@@ -275,20 +296,20 @@ def create(self, number: str, description: str, test_date: str, campaign_id: str
         body = dict(
             number=number,
             description=description,
-            type="Floater Test",
+            type=type,
```

Comment:
we shouldn't be able to override the test type. FloaterTestAPI.create() should always create a floater test